soc/espressif/esp32/mcuboot.ld

```diff
@@ -65,9 +64,6 @@ SECTIONS
     *libkernel.a:device.*(.literal .text .literal.* .text.*)
     *libkernel.a:timeout.*(.literal .text .literal.* .text.*)
 
-    *(.literal.bootloader_mmap .text.bootloader_mmap)
```

Comment:
aren't those used in the `bootloader_init()`

Reply:
This is not necessary in mcuboot booting context. Tested with all SoCs.